ui/src/widgets/bottom_statusbar.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2025 The Android Open Source Project
+//
```

Comment:
This isn't really a widget insofar that it cannot really be used any where else. I'd just move this into frontend/